roles/contrail_st/tasks/main.yaml

```diff
@@ -0,0 +1,3 @@
+---
+- name: Sanitize Schema Transformer process.
+  include_tasks: schema_transformer_sanity.yaml
```

Comment:
Doesn't make much sense to create a dedicated `schema_transformer_sanity.yaml` just to have it imported in one place. Move its contents here?

Reply:
Makes sense, will do this in a later commit, since its just cosmetics change ?

Reply:
Yes